### PR TITLE
ACBS

### PR DESCRIPTION
**Pull request by abhi-markan**
This commit deals with various ACBS bug patches on both deal and facilities records level.

### Deal Records
1. Investor record expiration date - ACBS bug raised with Mulesoft team, a temporary patch has been applied as per the recommendation by the Mulesoft support.
2. Deal currency - `GBP` returned as default currency when an unexpected object is returned.

### Facility Records
1. Master - Probability of default, an optional GEF field has been added.
2. Covenant - Maximum liability has been changed from UKEF exposure value to total facility value.
3. Helper functions - Introduction of following two helper functions to improve code readability and refactorization: 
    3.1. getCoverStartDate - Returns facilities cover start date
    3.2. hasFacilityBeenIssued - Returns whether facility has been issued

### CODE REVIEW

File: azure-functions/acbs-function/mappings/facility/facility-covenant.js [hunk 1]
```diff
@@ -26,7 +25,7 @@ const facilityCovenant = (deal, facility, covenantType) => {
     facilityIdentifier: facility.facilitySnapshot.ukefFacilityId.padStart(10, 0),
     portfolioIdentifier: CONSTANTS.FACILITY.PORTFOLIO.E1,
     covenantType,
-    maximumLiability: helpers.getMaximumLiability(facility.facilitySnapshot),
+    maximumLiability: Number(facility.facilitySnapshot.value),
```
Comment (ttbarnes) [inline azure-functions/acbs-function/mappings/facility/facility-covenant.js:28]: is `helpers.getMaximumLiability` no longer used or needed now?

Comment (abhi-markan) [reply azure-functions/acbs-function/mappings/facility/facility-covenant.js:28]: No longer needed due to AC.

**Review by IgnatG — APPROVED** [review]
LGTM

File: azure-functions/acbs-function/mappings/facility/facility-covenant.js (hunk 1 above)
Comment (ttbarnes) [reply azure-functions/acbs-function/mappings/facility/facility-covenant.js:28]: Just wondering if it's worth keeping the helper function or if we should remove? Happy either way 👍

**Pull request merged**